renovate.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:best-practices", ":disableDependencyDashboard", ":rebaseStalePrs"],
```

Default Presetで設定されているものは指定しなくてOKです

```suggestion
  "extends": ["config:best-practices", ":disableDependencyDashboard"],
```

ref: [Default Presets - Renovate Docs](https://docs.renovatebot.com/presets-default/#rebasestaleprs)